### ISSUE TITLE
Known Bug: AFT 1.6.2 and older - AWS EIP Allocation for NAT Gateway Fails

### ISSUE THREAD

**Issue by sirius-kris-leist** [aws-ia/terraform-aws-control_tower_account_factory#236]
**Terraform Version & Prov:**
terraform_version = "0.15.5"

**AFT Version:**
1.6.2

**Terraform Version & Provider Versions**
Please provide the outputs of `terraform version` and `terraform providers` from within your AFT environment

`terraform version`
```
Terraform v1.2.6
on darwin_arm64
+ provider registry.terraform.io/hashicorp/archive v2.2.0
+ provider registry.terraform.io/hashicorp/aws v4.29.0
+ provider registry.terraform.io/hashicorp/local v2.2.3
+ provider registry.terraform.io/hashicorp/random v3.4.2
+ provider registry.terraform.io/hashicorp/time v0.8.0
```

`terraform providers`
```
      source                = "hashicorp/aws"
      version               = ">= 4.9.0"
```

**Bug Description**
The eip addresses for NAT Gateways fails to provision with an error message about Classic VPC no longer being supported.  This can be fixed adding in vpc=true as noted below in:  modules/aft/modules/aft-account-request-framework/vpc.tf

```
resource "aws_eip" "aft-vpc-natgw-01" {
  vpc = true
}
```

```
resource "aws_eip" "aft-vpc-natgw-02" {
  vpc = true
}
```

**To Reproduce**
1. Deploy AFT into a greenfield Control Tower Environment.

**Expected behavior**
EIP's for NAT Gateways should be provisioned without error.

**Related Logs**
n/a

**Additional context**
n/a

**Comment by jodgey** [comment]
I am have the same issue. I am seeing the following error:

```
Do you want to perform these actions in workspace "tfcloud-ws-aws-aft"?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: 
module.aft.module.aft_ssm_parameters.aws_ssm_parameter.github_enterprise_url: Modifying... [id=/aft/config/vcs/github-enterprise-url]
module.aft.module.aft_account_request_framework.aws_eip.aft-vpc-natgw-01: Creating...
module.aft.module.aft_ssm_parameters.aws_ssm_parameter.github_enterprise_url: Modifications complete after 0s [id=/aft/config/vcs/github-enterprise-url]
╷
│ Error: with the retirement of EC2-Classic no new non-VPC EC2 EIPs can be created
│ 
│   with module.aft.module.aft_account_request_framework.aws_eip.aft-vpc-natgw-01,
│   on .terraform/modules/aft/modules/aft-account-request-framework/vpc.tf line 170, in resource "aws_eip" "aft-vpc-natgw-01":
│  170: resource "aws_eip" "aft-vpc-natgw-01" {}
│ 
╵
╷
│ Error: with the retirement of EC2-Classic no new non-VPC EC2 EIPs can be created
│ 
│   with module.aft.module.aft_account_request_framework.aws_eip.aft-vpc-natgw-02,
│   on .terraform/modules/aft/modules/aft-account-request-framework/vpc.tf line 172, in resource "aws_eip" "aft-vpc-natgw-02":
│  172: resource "aws_eip" "aft-vpc-natgw-02" {}
│ 
╵
Operation failed: failed running terraform apply (exit 1)
```

**Comment by stumins** [comment]
Hey @sirius-kris-leist @jodgey,

This for reporting this bug. We're working on a hotfix to resolve this issue.

**Comment by balltrev** [comment]
We've just released AFT [1.6.3](https://github.com/aws-ia/terraform-aws-control_tower_account_factory/releases/tag/1.6.3) which fixes this issue. Thanks again @jodgey @sirius-kris-leist!
